### DISCUSSION TITLE
Should snapshot dependencies web UI nest indirect dependencies?

### DISCUSSION THREAD

**Discussion by solvaholic** [community/community#35698] [Code Security] [answered]
👋 Hiyo! Looking at the beta feature _Snapshot dependencies associated with [@github/dependency-submission-toolkit](https://github.com/github/dependency-submission-toolkit/blob/main/%40github/dependency-submission-toolkit)_ in the **github/dependency-submission-toolkit** repository, I see it lists direct and indirect dependencies at the same level:
<https://github.com/github/dependency-submission-toolkit/network/dependencies>

> ![image](https://user-images.githubusercontent.com/14636658/194902210-f240a043-9d3f-4258-89aa-0ebe28ace566.png)

In contrast, the _Dependencies defined in [package.json](https://github.com/github/dependency-submission-toolkit/blob/main/package.json)_ section nests indirect dependencies under direct ones:

> ![image](https://user-images.githubusercontent.com/14636658/194902621-8213e707-5ad0-41d7-ac1a-6375996d090a.png)

Should snapshot dependencies also be nested, in the web UI?

**Comment by pcarlisle** [comment, marked as answer]
Yes, we plan to fix this so that indirect dependencies in snapshots have a nested display just like non-snapshot dependencies.
